lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -35,21 +35,28 @@
   - Gabriele Gerbino (@GGabriele)
 notes:
   - Tested against NXOSv 7.3.(0)D1(1) on VIRL
-  - The feature must be enabled with feature scp-server.
-  - If the file is already present (md5 sums match), no transfer will
-    take place.
+  - The feature must be enabled with feature scp-server if
+    the file is being pushed to the device. If file_pull
+    is True, feature scp-server is not needed.
+  - If the file is already present, no transfer will
+    take place if file_pull is False.
   - Check mode will tell you if the file would be copied.
 requirements:
```

Comment:
Can u update the requirements to indicate which are needed for `file_pull: True` vs. `file_pull: False`?